recipes/dbus/1.x.x/conanfile.py

```diff
@@ -135,8 +137,6 @@ def generate(self):
             tc.generate()
             cmake_deps = CMakeDeps(self)
             cmake_deps.generate()
-        pkg_config_deps = PkgConfigDeps(self)
-        pkg_config_deps.generate()
```

Comment:
I thought the CMake build system relied on finding some things through `pkg-config`, but then again maybe not. We'll find out.

Reply:
Only if with_systemd is enabled in CMake branching (so basically before 1.15 and only for Linux). But anyway pkgconf was missing in build requirements for this branching.